extensions/integration-tests/dockerContainers.ps1

```diff
@@ -0,0 +1,21 @@
+# Get where docker is installed
+$dockerPath = (where.exe docker) | Select-Object -First 1
+$dockerDirectory = $dockerPath.Substring(0, $dockerPath.Length - 21)
```

Comment:
Since we're in powershell, we get much better toys for stuff like this!

`$dockerDirectory = (Get-Command docker).Source`